spec/site_prism/page_objects/partials/header.rb

```diff
@@ -0,0 +1,7 @@
+module PageObjects
+  module Partials
+    class Header < SitePrism::Page
+      element :notifications_preference_link, "[data-qa=notifications-link]"
```

Comment:
We are trying to move away from using `data-qa` attributes for testing. 

What about something like
```suggestion
      element :notifications_preference_link, "a.govuk-header__link", text: "Notifications"
```

Reply:
Could use a similar lookup to test setting the active class styling